accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -0,0 +1,627 @@
+   :align: center
+ 
+
+Taxes
+~~~~~
+As part of the localization module the taxes are created automatically with their related
+financial account and configuration.
+
+.. image:: media/Chile08.png
+   :align: center
+
+Taxes Types
+***********
+Chile has several tax types, the most common ones are:
+
+- VAT. Is the regular VAT and it can have several rates. 
+- ILA (Impuesto a la Ley de Alcholes). Taxes for alcoholic drinks, it has a different rate.
+
+
+Fiscal Positions
+~~~~~~~~~~~~~~~~
+Based on the purchase transactions the VAT can have different affections, this will be done
+in Odoo using the default purchase fiscal positions. 
+
+
+Document Types
+~~~~~~~~~~~~~~
+In some Latin America countries, including Chile, some accounting transactions like invoices and
+vendor bills are classified by  document types defined by the government fiscal authorities (In
+Chile case: SII).
+
+The document type is an essential information that needs to be displayed in the printed reports and
+that needs to be easily identified, within the set of invoices as well of account moves.
+
+Each document type can have a unique sequence per journal where it is assigned. As part of the
+localization, the Document Type include the country on which the document is applicable and the data
+is created automatically when the localization module is installed.
+
+The information required for the document types is included by default so the user doesn't need to
+fill anything on this view:
+
+.. image:: media/Chile09.png
+   :align: center
+
+.. note::
+   There are several document types that are inactive by default but can be activated if needed.
+
+
+Use on Invoices
+***************
+The document type on each transaction will be determined by:
+
+- The Journal related to the Invoice, identifying if the journal uses documents.
+- Condition applied based on the type of Issues and Receiver (ex. Type of fiscal
+  regimen of the buyer and type of fiscal regimen of the vendor).
+  
+
+Journals
+--------
+
+Usage
+~~~~~
+
+Sales
+*****
+Sales Journals in Odoo usually represent a business unit or location, example:
+
+- Ventas Santiago.
+- Ventas Valparaiso.
+
+For the retail stores is common to have one journal per POS: 
+
+- Cashier 1.
+- Cashier 2.
+
+
+Purchases
+*********
+The transactions purchase can be managed with a single journal, but sometimes companies use
+more than one in order to handle some accounting transactions that are not related to vendor
+bills but can be easily register using this model, for example:
+
+- Tax Payments to government.
+- Employees payments.
+
+
+Configuration
+~~~~~~~~~~~~~
+When creating sales journals the next information must be filled in: 
+
+**Point of sale type**: If the Sales journal will be used for electronic documents the option
+Online must be selected, otherwise if the journal is used for invoices imported from a previous
+system or if you are using the SII portal “Facturación MiPyme“ you can use the option Manual. 
```

Comment:
```suggestion
**Point of sale type**: If the Sales journal will be used for electronic documents, the option
Online must be selected. Otherwise, if the journal is used for invoices imported from a previous
system or if you are using the SII portal “Facturación MiPyme“ you can use the option Manual. 
```